demo/demo.py

```diff
@@ -2,7 +2,11 @@
 import os
 import os.path as osp
 
+import cv2
+import decord
+import numpy as np
 import torch
+import webcolors
```

Comment:
`webcolors` should be put in requirements/optional.txt

Reply:
And how about put `decord` and `webcolor` into `try...except` to check whether users have installed them

Reply:
webcolors is a very small package (10KB) that map color names to RGB values. Decord is used frequently in mmaction2. Besides, it's easy to install both of them via pip. So that I think we can directly add them to setup.cfg instead